features/sketch/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id(libs.plugins.wire.android.library.get().pluginId)
+    id("org.jetbrains.kotlin.android")
+    alias(libs.plugins.ksp)
+}
+
+android {
+    namespace = "com.wire.android.feature.sketch"
```

Comment:
We could add the namespace definition [to the Plugin](https://github.com/wireapp/wire-android-reloaded/blob/52decf5ace84d1aad8f3fcb99c3e35decfb65e70/build-logic/plugins/src/main/kotlin/AndroidLibraryConventionPlugin.kt#L26), instead of copying/pasting to all modules:

```kotlin
    override fun apply(target: Project): Unit = with(target) {
        with(pluginManager){
            apply("com.android.library")
            apply("org.jetbrains.kotlin.android")
        }

        extensions.configure<LibraryExtension> {
            namespace = "com.wire.android.feature.${target.name}" // Namespace generated automatically based on the name of the module
        }
    }
```

Reply:
There was no copy paste :D the script actually is taking care of this, but anyway I think doing it like suggested ends being a better approach

Reply:
Yes yes. I wasn't clear when I said copy/paste. I know it's automated, but it's still copy/pasted. It's great to create, but not so easy to maintain.

For instance, this `namespace` used to be defined within the `Manifest.xml`. Since AGP 8.0.0, it's defined in Gradle like we're doing.

If the Android Gradle Plugin changes again and the way it is defined changes, we'd need to go and change all the modules.